src/main/java/github/SsItAcademy/PrimeDigits/myThread.java

```
@@ -0,0 +1,40 @@
+package github.SsItAcademy.PrimeDigits;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class myThread extends Thread {
+    private int a;
+    private int b;
+    static List<Integer> numsList = new ArrayList<Integer>();
+
+    myThread(int a, int b) {
+        this.a = a;
+        this.b = b;
+    }
+
+    @Override
+    public void run() {
+        try {
+            Thread.sleep(1000);
+        }catch (InterruptedException e){}
```

Don`t use empty catches.